src/widgets/SimpleTable.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable react/forbid-prop-types */
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { View, FlatList, TouchableOpacity } from 'react-native';
+import Cell from './DataTable/Cell';
+import { dataTableStyles } from '../globalStyles/index';
+import { HeaderCell, HeaderRow } from './DataTable/index';
+
+/**
+ * Simple table component for rendering a large list of un-changing data.
+ * Only offers a selection feature.
+ *
+ * Usage:
+ * Pass an array - data: [ { id, .. }, { id, .. }, .. ]
+ * Also an array - columns: [ { key, alignText, title } ] (example: getColumns.js)
+ * Such that each column.key is a field within an object in the data array.
+ *
+ * selectRow should be a function: (rowId) => { ... }
+ *
+ * extraData, an object with the shape: {rowId1: [bool], rowId2: [bool], ... } used
+ * to trigger styling effects on selected rows.
+ */
+export const SimpleTable = React.memo(({ data, columns, selectRow, extraData }) => {
+  const {
+    cellText,
+    cellContainer,
+    selectedRow: selectedRowStyle,
+    alternateRow: alternateRowStyle,
+    row: basicRowStyle,
+    headerRow,
+    headerCells,
+  } = dataTableStyles;
+
+  const renderCells = useCallback(rowData => {
+    const { id } = rowData;
+
+    return columns.map(({ key, width, alignText }, index) => (
+      <Cell
+        value={rowData[key]}
+        viewStyle={cellContainer[alignText]}
+        textStyle={cellText[alignText]}
+        isLastCell={index === columns.length - 1}
+        width={width}
+        key={`${id}${key}`}
+      />
+    ));
+  }, []);
```

Comment:
` [rowData]`? or doesn't matter? It's not likely to re-render anyway is it?

Reply:
`rowData` is a parameter not a free variable

Reply:
will take your word for it ( which means, I dunno. seems to work fine, so let's just leave it )